db/migrate/20210610150945_create_deltas.rb

```diff
@@ -0,0 +1,13 @@
+Sequel.migration do
+  change do
+    create_table :deltas do
+      primary_key :id
+      String  :goods_nomenclature_item_id
+      String  :goods_nomenclature_sid
```

Comment:
Do we want to make this match the type of the other table to make it easy to setup relationships on the delta model at some later stage/be consistent